lua/dyn_menu.lua

```diff
@@ -54,11 +64,11 @@ function build_track_title(track, prefix, filename)
     if track['demux-fps'] then h(string.format('%.5g fps', track['demux-fps'])) end
     if track['audio-channels'] then h(track['audio-channels'] .. ' ch') end
     if track['demux-samplerate'] then h(string.format('%.3g kHz', track['demux-samplerate'] / 1000)) end
-    if track['demux-bitrate'] then h(string.format('%.3g kpbs', track['demux-bitrate'] / 1000)) end
+    if track['demux-bitrate'] then h(string.format('%.3g kbps', track['demux-bitrate'] / 1000)) end
     if track.forced then h('forced') end
+    if track.default then h('default') end
```

Comment:
Unrelated change

Reply:
Split into a separate commit.

Reply:
Why do you want to revert it? I do think `(*)` is more readable than `default`

Reply:
Although the `default` property are also printed in this way of `(*)` in the console, these properties of `forced` and `external` are not displayed in the console.
When rendering in the menu, that is not coordinated to only specialize the default property because we display all of them simultaneously.
If you have a way to handle `forced` and `external` as well, I don't object to retaining the current behavior.

Reply:
Remove the commit, there is a issue with the idea here.

Reply:
You may want to check mpv's print_stream function:

https://github.com/mpv-player/mpv/blob/41259db9523bede8c656c7a8664484e0e4befa02/player/loadfile.c#L241-L298

Reply:
That's exactly why I added a new commit. The issue here is that the meaning of abbreviations is unclear, and more than one person on the mpv issue is already confused about the meaning and function of the `[F]` button on osc. The `(*)` Similarly, there will always be people questioning its meaning. 
However, I confirm that there are scenarios where `forced` and `default` or `external` and `default` coexist, so omitting a few characters makes sense. Therefore, in the end, I chose to keep the `(*)`.